### PR TITLE
improved dockerfile

### LINKED ISSUE #964: Image build process is extremely slow
The blobber and validator image build process is slow, it takes about 20m to build blobber, 23m for validator
image

### PR DESCRIPTION
**Pull request by Manali-Jain-squareops**
### Tests
Tasks to complete before merging PR:
- [ ]  Ensure system tests are passing. If not [Run them manually](https://github.com/0chain/blobber/actions/workflows/system_tests.yml) to check for any regressions :clipboard:
- [ ]  Do any new system tests need added to test this change? do any existing system tests need updated? If so create a PR at [0chain/system_test](https://github.com/0chain/system_test)
- [ ]  Merge your system tests PR to master AFTER merging this PR

### Associated PRs (Link as appropriate):
- 0chain:
- gosdk:
- system_test:
- zboxcli:
- zwalletcli:
- Other: ...

### CODE REVIEW

File: docker.local/base.Dockerfile [hunk 1]
```diff
@@ -5,37 +5,26 @@ LABEL zchain="blobber"
 # https://mirrors.alpinelinux.org/
 # RUN sed -i 's/dl-cdn.alpinelinux.org/mirrors.aliyun.com/g' /etc/apk/repositories
```
Comment (boddumanohar) [inline docker.local/base.Dockerfile]: please remove the commented code.

**Review by boddumanohar — APPROVED** [review]
LGTM

File: docker.local/base.Dockerfile (hunk 1 above)
Comment (Manali-Jain-squareops) [reply docker.local/base.Dockerfile]: Done

**Comment by peterlimg** [comment]
@Manali-Jain-squareops can we do the base image build the same way as 0chain to build miner/sharder. I find the miner/sharder could be done the base image build in 1s by using the github cache, which would save us a lot time.

**Pull request merged**